SECURITY.md

```
@@ -0,0 +1,172 @@
+| ------- | ------------------ |
+| 1.0.x   | ✅ Full support    |
+| 0.5.x   | ✅ Security fixes  |
+| 0.4.x   | ❌ End of life     |
+| < 0.4   | ❌ No support      |
+
+## 🚨 Reporting a Vulnerability
+
+### **How to Report**
+If you discover a security vulnerability in RudeBase3D, please report it responsibly:
+
+**📧 Email**: security@no-hands-company.com
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)